src/components/text-editor/examples/text-editor-with-inline-images-file-storage.tsx

```
@@ -0,0 +1,116 @@
+
+    @State()
+    private uploadImageFails: boolean = false;
+
+    public render() {
+        return [
+            <limel-text-editor
+                value={this.value}
+                onChange={this.handleChange}
+                onImagePasted={this.handleImagePasted}
+                onImageRemoved={this.handleImageRemoved}
+            />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)